assets/src/edit-story/elements/text/edit.js

```diff
@@ -335,12 +338,19 @@ function TextEdit({
   const hasHighlightBackgroundTextMode =
     backgroundTextMode === BACKGROUND_TEXT_MODE.HIGHLIGHT;
 
+  const wrapperProps = {
+    backgroundColor: wrapperBackgroundColor,
+  };
+  if (borderRadius) {
+    wrapperProps.borderRadius = borderRadius;
+  }
```

Comment:
If not set, is it a problem if passed to the component? Could you not just do `borderRadius={borderRadius}` in the render below, and it would work as expected both if set and if not?